stock_vlm_mgmt/static/src/js/stock_location_tray.js

```diff
@@ -0,0 +1,284 @@
+/* Copyright 2019 Camptocamp SA
```

Comment:
what about this? :thinking:

Reply:
This is the only one we have reused and we have kept the attribution as it should. The rest of the models and code are new.